src/contents/all/components/app/App.tsx

```diff
@@ -69,11 +69,19 @@ const App = () => {
 
 
   const updateLogStatus = target => {
-
-    const newLog = { ...log, status: target.key }
-    setLog(newLog)
-    setMessage(statusMessage[target.key])
-    chrome.runtime.sendMessage({ message: "log_update", log: newLog })
+    if (target.key == "delete") {
```

Comment:
originally was going to do this in its own method (`const deleteLog =>`) but couldn't find a clean way to call new function in html dropdown. this is something i can look into later!

Reply:
for sure, sgtm!